cmd/process-agent/command/main_common.go

```diff
@@ -296,13 +301,7 @@ func initMisc(deps miscDeps) error {
 	// appCtx is a context that cancels when the OnStop hook is called
 	appCtx, stopApp := context.WithCancel(context.Background())
 	deps.Lc.Append(fx.Hook{
-		OnStart: func(startCtx context.Context) error {
-
-			err := manager.ConfigureAutoExit(startCtx, deps.Config)
```

Comment:
Did you confirm with the process agent team that using `startCtx` was a mistake?

Reply:
I asked @just-chillin, who wrote this piece of code. I didn't get her answer yet.

Reply:
I believe it is a mistake, thanks for catching it!